### ISSUE TITLE
Make Windows Pop-in

### ISSUE THREAD

**Issue by simlu** [simlu/voxelshop#83]
Make windows "poping in" again from the side. Not sure why I've disabled it, but there was a reason.

**Comment by simlu** [comment]
This should help a lot with people being overwhelmed initially by too many windows....

**Comment by simlu** [comment]
Should some frames be hidden by default?

**Comment by MrTopom** [comment]
Maybe provide some topical layout like in dev environnement with more or less windows.

**Comment by simlu** [comment]
Yeah, same as JetBrains does it. Should make the view way cleaner.
Reference: http://twimgs.com/ddj/images/article/2012/0412/jetbrains.gif
